models/spring-ai-azure-openai/src/main/java/org/springframework/ai/azure/openai/AzureOpenAiChatClient.java

```diff
@@ -15,10 +15,7 @@
  */
 package org.springframework.ai.azure.openai;
 
-import java.util.Collections;
```

Comment:
Wildcard imports such as import java.util.* or import static org.assertj.core.api.Assertions.* are forbidden, even in test code: https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements

Reply:
removed all star imports from all sources